pkg/services/thumbs/service.go

```diff
@@ -230,22 +257,30 @@ func (hs *thumbService) StartCrawler(c *models.ReqContext) response.Response {
 	if cmd.Mode == "" {
 		cmd.Mode = CrawlerModeThumbs
 	}
-	msg, err := hs.renderer.Start(c, cmd.Mode, cmd.Theme, models.ThumbnailKindDefault)
+
+	go hs.runOnDemandCrawl(context.Background(), cmd.Theme, cmd.Mode, models.ThumbnailKindDefault, rendering.AuthOpts{
+		OrgID:   c.OrgId,
+		UserID:  c.UserId,
+		OrgRole: c.OrgRole,
+	})
+
+	status, err := hs.renderer.Status()
```

Comment:
`Status()` call doesn't make sense in HA environment, I think this function can just return empty `201` and make client wait for updates from grafana live